Client/Frontend/Browser/MailProviders.swift

```diff
@@ -7,75 +7,115 @@ import Shared
 
 // mailto headers: subject, body, cc, bcc
 
+enum MailProviderEmailFormat {
+    case standard
+    case userAndPassword // used for Protonmail
```

Comment:
note: lets change this to `protonMailFormat` as all this work is for proton mail